pom.xml

```diff
@@ -129,7 +129,7 @@
         <dependency>
             <groupId>org.gridsuite</groupId>
             <artifactId>gridsuite-network-modification</artifactId>
-            <version>${network-modification.version}</version>
+            <version>0.3.0-SNAPSHOT</version>
```

Comment:
to be removed

Reply:
Yes but I keep it here to simplify my test for now.

I precised in the PR description that 2 changes have to be removed before the merging. This is one of them.